doc/faq.rst

```diff
@@ -165,6 +165,66 @@ Using Spyder
    Click :guilabel:`Restart kernel` in the :guilabel:`Consoles` menu for this change to take effect.
 
 
+.. dropdown:: Q: How do I install Python packages to use within Spyder if downloaded Spyder from the standalone installers?
+   :container: + dropdown-id-using-packages-installer
+
+   If you want to use other modules in Spyder that don't come with our installer, you need to install `Miniconda`_ (**Only if you don't have Anaconda or Miniconda yet!**). For Spyder to recognize it, the installation should be done in one of the default paths. See the table below:
```

Comment:
```suggestion
   If you want to use other modules in Spyder that don't come with our installer, you need to install `Miniconda`_ (**only if you don't have Anaconda or Miniconda yet!**). For Spyder to recognize it, the installation should be done in one of the following default paths:
```

- There's no need to start a sentence capitalized when it's in parenthesis.
- I think the my second suggestion add better rhythm to the sentence.